src/components/Popup/Popup.tsx

```diff
@@ -96,56 +109,97 @@ export default class Popup extends UIComponent<Extendable<IPopupProps>, IPopupSt
     as: Portal,
     align: 'start',
     position: 'above',
+    accessibility: PopupBehavior as Accessibility,
   }
 
-  public state = { triggerRef: undefined }
+  public state = { triggerRef: undefined, popupOpened: false }
+
+  actionHandlers: AccessibilityActionHandlers = {
+    performClick: e => this.onTriggerClick(e),
+    closeAndFocusTrigger: e => this.handlePopupState(e, false, this.focusTrigger),
+  }
 
   public renderComponent({
     ElementType,
     classes,
+    accessibility,
     rest,
     rtl,
   }: IRenderResultConfig<IPopupProps>): React.ReactNode {
-    const { children, trigger } = this.props
+    const { children, trigger, position, align } = this.props
 
     return (
       <ElementType
         className={classes.root}
         {...rest}
+        open={this.state.popupOpened}
```

Comment:
even when it was just about `trigger` and `triggerRef` props being defined for the `ElementType` component, we already had the problem related to restricted options for `as` values. Now the problem is worse - we definitely need to address this in the nearest future, as there are some options for that

Reply:
Totally agree with that, same as passing accessibility attributes for the ```trigger``` element